src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py

```diff
@@ -428,13 +429,14 @@ def __call__(
             prompt_embeds = torch.cat([negative_prompt_embeds, prompt_embeds], dim=0)
 
         # 4. Prepare timesteps
+
+        sigmas = np.linspace(1.0, 1 / num_inference_steps, num_inference_steps)
```

Comment:
I added this just to get 1:1 with the original implementation, I don't think it's needed here 
cc @cloneofsimo 

without this line
![yiyi_test_4_out](https://github.com/huggingface/diffusers/assets/12631849/9a8de55b-8413-4eed-85e7-85798c2b6bfd)

with this line
![yiyi_test_4_out_yiyi](https://github.com/huggingface/diffusers/assets/12631849/c49b6250-3dca-40ff-b81d-3ee75be5ee2c)